messaging-activemq/src/main/resources/org/wildfly/extension/messaging/activemq/LocalDescriptions.properties

```diff
@@ -194,6 +194,7 @@ connection-factory.scheduled-thread-pool-max-size=The scheduled thread pool max
 connection-factory.thread-pool-max-size=The thread pool max size.
 connection-factory.transaction-batch-size=The transaction batch size.
 connection-factory.use-global-pools=True to use global pools.
+connection-factory.use-topology-for-load-balancing=Use cluster topology to connect to it
```

Comment:
"Whether to use topology information from the cluster to connect to it. If {@code false}, <insert what>